lib/asciidoctor.rb

```diff
@@ -501,7 +500,7 @@ class << self; self; end.send :attr_accessor, key
     #                collapsing the line breaks and indentation to
     #                a single space.
     #
-    AttributeEntryRx = /^:(!?\w.*?):(?:#{CG_BLANK}+(.*))?$/
+    AttributeEntryRx = /^:(!?\w.*?):(?:\s+(.*))?$/
```

Comment:
We probably should use `[ \t]` here anyway since that's what we use for other line-oriented regexps. Better to be consistent.